common/src/main/java/com/microsoft/identity/common/internal/controllers/BrokerMsalController.java

```diff
@@ -800,6 +803,71 @@ public void putValueInSuccessEvent(final @NonNull ApiEndEvent event, final @NonN
                 });
     }
 
+
+    /**
+     * Checks if QR + PIN authorization is available.
+     *
+     * @return true if if QR + PIN authorization is available. False otherwise.
+     */
+    @Override
+    public boolean isQrPinAvailable() throws BaseException {
+        final String methodTag = TAG + ":isQrPinAvailable";
+        return mBrokerOperationExecutor.execute(
```

Comment:
Should we make this call to always go to Authenticator from the Msal side, instead of the active Broker?
and do the checks for Authenticator being installed on the Msal layer, instead of Broker layer?

Since this code is in common, OneMsal should also work.